storybook/src/components/PageHeading/PageHeading.docs.mdx

```diff
@@ -3,6 +3,12 @@
 import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/addon-docs/blocks";
 import * as PageHeadingStories from "./PageHeading.stories.tsx";
 import README from "../../../../packages/css/src/components/page-heading/README.md?raw";
+import { StatusBadge } from "../../docs/components/StatusBadge";
+
+<StatusBadge
```

Comment:
Shouldn't we just remove the entire docs page here? That would reduce the chance of people using it accidentally.